src/Geometry/BoundaryTriangulations.jl

```diff
@@ -162,9 +162,8 @@ function BoundaryTriangulation(
 end
 
 function BoundaryTriangulation(
-  model::DiscreteModel,
-  face_to_bgface::AbstractVector{<:Integer})
-  BoundaryTriangulation(model,face_to_bgface,Fill(1,num_facets(model)))
+  model::DiscreteModel, face_to_bgface::AbstractVector{<:Integer}, lcell::Integer=1)
```

Comment:
I guess this change is something not related to the ambiguity, but that was somehow missing in the previous version of Gridap?

Reply:
Yeah, I just saw we had the same signature for bools so I guessed it does not harm to have this one as well.